cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-rococo/src/tests/teleport.rs

```diff
@@ -558,30 +559,19 @@ fn teleport_to_other_system_parachains_works() {
 /// (using native reserve-based transfer for fees)
 #[test]
 fn bidirectional_teleport_foreign_assets_between_para_and_asset_hub() {
-	let ah_as_seen_by_penpal = PenpalA::sibling_location_of(AssetHubRococo::para_id());
+	// Init values for Parachain
+	let fee_amount_to_send: Balance = ASSET_HUB_ROCOCO_ED * 10000;
 	let asset_location_on_penpal = PenpalLocalTeleportableToAssetHubV3::get();
 	let asset_id_on_penpal = match asset_location_on_penpal.last() {
 		Some(v3::Junction::GeneralIndex(id)) => *id as u32,
 		_ => unreachable!(),
 	};
-	let asset_owner_on_penpal = PenpalASender::get();
-	let foreign_asset_at_asset_hub_rococo =
-		v3::Location::new(1, [v3::Junction::Parachain(PenpalA::para_id().into())])
-			.appended_with(asset_location_on_penpal)
-			.unwrap();
-	super::penpal_create_foreign_asset_on_asset_hub(
-		asset_id_on_penpal,
-		foreign_asset_at_asset_hub_rococo,
-		ah_as_seen_by_penpal.clone(),
-		false,
-		asset_owner_on_penpal,
-		ASSET_MIN_BALANCE * 1_000_000,
-	);
```

Comment:
Please add a comment wherever this is removed from, explaining that the foreign asset is already created and some accounts prefunded in genesis, otherwise the test becomes confusing to a new comer.